src/sbt-test/sbtbiopet/developtool/project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 sys.props.get("plugin.version") match {
   case Some(x) => addSbtPlugin("com.github.biopet" % "sbt-biopet" % x)
-  case _ => addSbtPlugin("com.github.biopet" % "sbt-biopet" % "0.2-SNAPSHOT")
+  case _ => addSbtPlugin("com.github.biopet" % "sbt-biopet" % "0.4-SNAPSHOT")
```

Comment:
Can this be done with a variable? This seems likely to break in next versions.

Reply:
fixed